pkg/codegen/testing/test/testdata/jumbo-resources/schema.json

```diff
@@ -773,6 +773,9 @@
         },
         "bar256": {
           "type": "string"
+        },
+        "default": {
```

Comment:
Curious why add this one? To test name mangling?

Reply:
yes, I wanted to test the `default` -> `default_` and that one or the other will appear in all the correct places